cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/leveldb/LevelDBStreamAdmin.java

```diff
@@ -41,8 +45,9 @@ public LevelDBStreamAdmin(CConfiguration conf, LevelDBOrderedTableService servic
   @Override
   public String getActualTableName(QueueName queueName) {
     if (queueName.isStream()) {
-      // <cdap namespace>.system.stream.<account>.<stream name>
-      return getTableNamePrefix() + "." + queueName.getFirstComponent();
+      //TODO: namespace the tableName (have to modify tableNamePrefix)
+      // <cdap namespace>.system.stream.<stream name>
+      return getTableNamePrefix() + "." + queueName.getSecondComponent();
```

Comment:
So the first component is already *namespace* but its always *default* right now?